docs/customisation/configuration.md

```diff
@@ -18,3 +20,20 @@ The properties used by Inji are:
 * `mosip.inji.vcDownloadPoolInterval(6000)`: The waiting interval between each retry.
 * `mosip.inji.audience(ida-binding)`: This is used to generate the JWT token which will be passed during online login.
 * `mosip.inji.issuer(residentapp)`: This is used to generate the JWT token which will be passed during online login.
+* `mosip.inji.warningDomainName(https://${mosip.api.public.host})`: This is the domain used to access all Apis.
+* `mosip.inji.aboutInjiUrl(https://docs.mosip.io/inji)`: This is the url for Inji documentation used in About Inji page.
+* `mosip.inji.minStorageRequiredForAuditEntry(1)`: This is threshold(minimum storage space required) in MB for storing audit entries.
+* `mosip.inji.minStorageRequired(5)`: This is threshold(minimum storage space required) in MB for storing downloaded / received vc.
+
+
+## Configuration for Issuers
+
+The configurable properties for supporting issuers can be found at [mimoto-issuers-config.json](https://github.com/mosip/mosip-config/blob/qa-inji/mimoto-issuers-config.json). This property file is maintained as one for each deployment environment. On [this](https://github.com/mosip/mosip-config) repository, each environment configuration is placed in a corresponding branch specific to that environment.
+
+> Refer to [mimoto-issuers-config.json](https://github.com/mosip/mosip-config/blob/collab/mimoto-issuers-config.json) of Collab environment.
```

Comment:
As of today, this is not valid link. It will be valid once release changes deployed to collab env